README.md

```diff
@@ -98,15 +98,131 @@ This release extends support for RHEL v8.5.
+| gfx90a:xnack- | MI210      | Aldebaran       | TBA                                               |  ✅   |    ✅    |   ✅    |    ✅    |    ✅    |     ✅     |     ✅     |     ✅     |     ✅      |    ✅    |
+| gfx1010       | Navi       | Navi 10 XT      | AMD Radeon™ PRO 5600M                             |  ❌   |    ✅    |   ❌    |    ❌    |    ❌    |     ✅     |     ❌     |     ❌     |     ❌      |    ∅    |
+| gfx1010       | Navi       | Navi 10 XLE     | AMD Radeon™ PRO 5600 XT                           |  ❌   |    ✅    |   ❌    |    ❌    |    ❌    |     ✅     |     ❌     |     ❌     |     ❌      |    ∅    |
+| gfx1010       | Navi       | Navi 10 XL      | AMD Radeon™ RX 5700                               |  ❌   |    ✅    |   ❌    |    ❌    |    ❌    |     ✅     |     ❌     |     ❌     |     ❌      |    ∅    |
+| gfx1010       | Navi       | Navi 10 XT      | AMD Radeon™ RX 5700 XT                            |  ❌   |    ✅    |   ❌    |    ❌    |    ❌    |     ✅     |     ❌     |     ❌     |     ❌      |    ∅    |
+| gfx1012       | Navi       | Navi 14 XT      | AMD Radeon™ RX 5500                               |  ❌   |    ✅    |   ❌    |    ❌    |    ❌    |     ✅     |     ❌     |     ❌     |     ❌      |    ∅    |
+| gfx1012       | Navi       | Navi 14 XTX     | AMD Radeon™ RX 5500 XT                            |  ❌   |    ✅    |   ❌    |    ❌    |    ❌    |     ✅     |     ❌     |     ❌     |     ❌      |    ∅    |
+| gfx1030       | Big Navi   | Navi 21         | AMD Radeon™ RX 6800                               |  ✅   |    ✅    |   ✅    |    ✅    |    ✅    |     ✅     |     ✅     |     ✅     |     ✅      |    ∅    |
+| gfx1030       | Big Navi   | Navi 21         | AMD Radeon™ RX 6800 XT                            |  ✅   |    ✅    |   ✅    |    ✅    |    ✅    |     ✅     |     ✅     |     ✅     |     ✅      |    ∅    |
+| gfx1030       | Big Navi   | Navi 21         | AMD Radeon™ RX 6900 XT                            |  ✅   |    ✅    |   ✅    |    ✅    |    ✅    |     ✅     |     ✅     |     ✅     |     ✅      |    ∅    |
+| gfx1030       | Big Navi   | Navi 21         | AMD Radeon™ RX 6950 XT                            |  ✅   |    ✅    |   ✅    |    ✅    |    ✅    |     ✅     |     ✅     |     ✅     |     ✅      |    ∅    |
+| gfx1030       | Big Navi   | Navi 21         | AMD Radeon™ PRO V620                              |  ✅   |    ✅    |   ✅    |    ✅    |    ✅    |     ✅     |     ✅     |     ✅     |     ✅      |    ∅    |
+| gfx1030       | Big Navi   | Navi 21         | AMD Radeon™ PRO W6800                             |  ✅   |    ✅    |   ✅    |    ✅    |    ✅    |     ✅     |     ✅     |     ✅     |     ✅      |    ∅    |
+
+Legend of this table:
+
+| Symbol |           Meaning           |
+|:------:|:---------------------------:|
+|   ∅    |    No required hardware     |
+|   ✅    |     Library targets GPU     |
+|   ❌    | Library omits targeting GPU |
+
+Sources of this table:
```

Comment:
Not that it will make updating the README easier, but I think it's a bit dangerous to link to the develop branch of all of these libraries. The develop branch will change in the future (e.g., they might remove support), while someone linking to a particular fixed version of the ROCm README may want to know what versions were checked to keep this table up to date (i.e., you may want to link to a pinned commit for each of the links below).

Reply:
> e.g., they might remove support

This motivates me to keep the dangerous link to the development branch to `prevent support removal` and have longer-term support, or at list it just-works™.

Reply:
Decisions on library support strategies are made independent of this particular README (i.e., their support drives this README, rather than the other way around), so I'm not sure that plan would leave us with much besides a once-again-outdated™ set of documentation.

Reply:
Feel free to invite me to the strategy meetings. :-)